tests/containers/docker_image.pm

```diff
@@ -37,13 +38,16 @@ sub run {
     scc_apply_docker_image_credentials() if (get_var('SCC_DOCKER_IMAGE'));
 
     for my $iname (@{$image_names}) {
+        record_info 'image name', '$iname';
         test_container_image(image => $iname, runtime => $runtime);
-        build_container_image(image => $iname, runtime => $runtime);
-        if (check_os_release('suse', 'PRETTY_NAME')) {
+        build_container_image(image => $iname, runtime => $runtime) if is_sle_container_supported;
+
+        if (check_os_release('suse', 'PRETTY_NAME') && is_sle_container_supported) {
```

Comment:
This by the way means that `if(is_sle_container_supported()){test_opensuse_based_image()};` which does not make a sense.

Reply:
let me take a look. i think i messed up here